Packs/SpurContextAPI/Integrations/SpurContextAPI/SpurContextAPI.yml

```diff
@@ -3,12 +3,22 @@ commonfields:
   id: SpurContextAPI
   version: -1
 configuration:
-- display: ""
+- display: "Server URL (e.g. https://api.spur.us/)"
+  name: base_url
+  type: 0
+  required: true
```

Comment:
i think it's not supposed to be required, since it has just been added, and it has a default value

Reply:
Changed to False.

Reply:
it's still true